scripts/test_unit.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
 # test_unit.sh
+
+source cmd/proxy/.env
+export TMPDIR=$(mktemp -d)
```

Comment:
I'm so sorry, I swear I'm not doing this on purpose, it's the end 2 end tests that use the temp directory. We don't need this at all. 

As well I was looking and there will be some other small changes when we switch to using `mktemp` in other scripts. I'll take care of that change later today.

So last change and we can merge. 😁